### PR TITLE
docs: increase specificity of property comment

### PR DESCRIPTION
**Pull request by DoctypeRosenthal**

### CODE REVIEW

**Comment by gsimone** [comment]
LGTM

**Review by gsimone — APPROVED** [review]

**Comment by gsimone** [comment]
@drcmda can you confirm this is an appropriate description?

**Comment by drcmda** [comment]
@DoctypeRosenthal almost, it's in pixels afaik https://github.com/pmndrs/react-three-fiber/blob/a9e09a31991ff7fb99292ecd68a8a700eaff4b0f/packages/fiber/src/core/events.ts#L124-L129

File: docs/react-three-fiber/API/events.mdx [hunk 1]
```diff
@@ -41,7 +41,7 @@ Also notice the `onPointerMissed` on the canvas element, which fires on clicks t
   ray: Ray                      // The ray that was used to strike the object
   camera: Camera                // The camera that was used in the raycaster
   sourceEvent: DomEvent         // A reference to the host event
-  delta: number                 // Initial-click delta
+  delta: number                 // Distance between mouse down and mouse up event in threejs units
```
Comment (drcmda) [inline docs/react-three-fiber/API/events.mdx:44]: "in pixels"

**Review by drcmda — CHANGES_REQUESTED** [review]

**Pull request merged**